Gemfile

```diff
@@ -0,0 +1,2 @@
+source "https://rubygems.org"

```

Comment:
Without them I have issue with scss-lint - this is the only reason I added them. Saw your comment on another PR: happy to gitignore them if don't need them anymore!